<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nurikabe Maker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets.qrc">
    <normaloff>:/icon/assets/icon.ico</normaloff>:/icon/assets/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="Board"/>
      </item>
      <item>
       <widget class="QLabel" name="console">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="statusView">
        <property name="font">
         <font>
          <family>Trebuchet MS</family>
          <weight>75</weight>
          <bold>true</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>CREATE</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="solvePuzzle">
        <property name="text">
         <string>Solve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generatePuzzle">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="checkPuzzle">
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="undoMove">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Undo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="redoMove">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>36</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Redo</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>564</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Change Mode</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionSolve"/>
    <addaction name="actionView"/>
   </widget>
   <widget class="QMenu" name="menuStates">
    <property name="title">
     <string>States</string>
    </property>
    <widget class="QMenu" name="menuLoad">
     <property name="title">
      <string>Load</string>
     </property>
     <addaction name="loadState_1"/>
     <addaction name="loadState_2"/>
     <addaction name="loadState_3"/>
    </widget>
    <widget class="QMenu" name="menuSave">
     <property name="title">
      <string>Save</string>
     </property>
     <addaction name="saveState_1"/>
     <addaction name="saveState_2"/>
     <addaction name="saveState_3"/>
    </widget>
    <addaction name="menuSave"/>
    <addaction name="menuLoad"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuStates"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icon/assets/load_file.svg</normaloff>:/icon/assets/load_file.svg</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icon/assets/exit_app.svg</normaloff>:/icon/assets/exit_app.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icon/assets/new_file.svg</normaloff>:/icon/assets/new_file.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icon/assets/save_file.svg</normaloff>:/icon/assets/save_file.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionSolve">
   <property name="text">
    <string>Solve</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>View</string>
   </property>
  </action>
  <action name="saveState_1">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icon/assets/save_file.svg</normaloff>:/icon/assets/save_file.svg</iconset>
   </property>
   <property name="text">
    <string>State 1</string>
   </property>
  </action>
  <action name="saveState_2">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icon/assets/save_file.svg</normaloff>:/icon/assets/save_file.svg</iconset>
   </property>
   <property name="text">
    <string>State 2</string>
   </property>
  </action>
  <action name="saveState_3">
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icon/assets/save_file.svg</normaloff>:/icon/assets/save_file.svg</iconset>
   </property>
   <property name="text">
    <string>State 3</string>
   </property>
  </action>
  <action name="loadState_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icon/assets/invalid.svg</normaloff>:/icon/assets/invalid.svg</iconset>
   </property>
   <property name="text">
    <string>State 1</string>
   </property>
  </action>
  <action name="loadState_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icon/assets/invalid.svg</normaloff>:/icon/assets/invalid.svg</iconset>
   </property>
   <property name="text">
    <string>State 2</string>
   </property>
  </action>
  <action name="loadState_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="assets.qrc">
     <normaloff>:/icon/assets/invalid.svg</normaloff>:/icon/assets/invalid.svg</iconset>
   </property>
   <property name="text">
    <string>State 3</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="assets.qrc"/>
 </resources>
 <connections/>
</ui>
